bluebottle/time_based/states/participants.py

```diff
@@ -494,3 +494,9 @@ class TeamScheduleParticipantStateMachine(ScheduleParticipantStateMachine):
 @register(PeriodicParticipant)
 class PeriodicParticipantStateMachine(RegistrationParticipantStateMachine):
     pass
+
+
+@register(PeriodParticipant)
```

Comment:
This should not be nessecary. This is only needed when something went wrong in the migration. By adding these line, we will hide the actual error for ourselves.

Reply:
deloitte-uk tenant was not working for me.
I'll remove this